tests/gfm_tasklist.rs

```diff
@@ -0,0 +1,68 @@
+// This file is auto-generated by the build script
+// Please, do not modify it manually
+
+extern crate pulldown_cmark;
+
+include!("normalize_html.rs.inc");
+
+
+    #[test]
+    fn gfm_tasklist_test_1() {
+        let original = r##"- [ ] foo
+- [x] bar
+"##;
+        let expected = r##"<ul>
```

Comment:
This test should expect list / list items with relevant classes.

```
<ul class="contains-task-list">
<li class="task-list-item"><input type="checkbox" id="" disabled="" class="task-list-item-checkbox">test item</li>
</ul>
```
This detail is missing from the GFM spec.  However there's no CSS way to locate the items and apply the style to get rid of the list item symbols.

https://github.github.com/gfm/#task-list-items-extension-